src/NSL/Action/Implementations/hubbardFermiAction.tpp

```diff
@@ -43,14 +43,21 @@ class HubbardFermionAction :
         // inverse temperature
 	    const Type beta;
 
+        // chemical potential
+        const Type mu;
+
```

Comment:
This made me remember that µ could potentially be complex.  That raises some issues below.